buttercup/cogs/history.py

```diff
@@ -261,6 +280,133 @@ async def _history(
             file=discord_file,
         )
 
+    def get_user_rate(self, user: str) -> pd.DataFrame:
+        """Get a data frame representing the transcription rate of the user.
+
+        :returns: The rate data of the user.
+        """
+        # First, get the ID of the user
+        user_response = self.blossom_api.get_user(user)
+        if user_response.status != BlossomStatus.ok:
+            raise BlossomException(user_response)
+
+        user_id = user_response.data["id"]
+
+        # Get all rate data
+        user_data = self.get_all_rate_data(user_id, "day")
+
+        # Add an up-to-date entry
+        today = datetime.now(tz=tzutc()).replace(
+            hour=0, minute=0, second=0, microsecond=0
+        )
+        if today not in user_data.index:
+            user_data.loc[today] = [0]
+
+        return user_data
+
+    @cog_ext.cog_slash(
+        name="rate",
+        description="Display the rate graph.",
+        options=[
+            create_option(
+                name="user_1",
+                description="The user to display the rate graph for.",
+                option_type=3,
+                required=False,
+            ),
+            create_option(
+                name="user_2",
+                description="The second user to add to the graph.",
+                option_type=3,
+                required=False,
+            ),
+            create_option(
+                name="user_3",
+                description="The third user to add to the graph.",
+                option_type=3,
+                required=False,
+            ),
+        ],
+    )
+    async def _rate(
+        self,
+        ctx: SlashContext,
+        user_1: Optional[str] = None,
+        user_2: Optional[str] = None,

```

Comment:
Is there an option for list arguments? If so, I would rather see us use a list with the parameter validation that checks whether the list contains 3 or fewer items.

Reply:
Unfortunately, I don't think there is.

Reply:
Nevermind, I found a way: You can now simply provide all usernames separated by a space in the first command. I actually like this a lot more, because it's easier to type and we can then use a second parameter to provide an optional time range. I was already wondering how to best include that option.

Thanks for the suggestion!